contracts/governance_admin/src/execute/resume_join_pool.rs

```diff
@@ -198,12 +194,31 @@ pub fn execute_resume_join_pool(
         );
 
         // add an event
-        event = event.add_attribute(
-            "reward_schedules_creation_request_id",
-            reward_schedules_creation_request_id.to_string(),
-        );
+        event = event
+            .add_attribute(
+                "reward_schedules_creation_request_id",
+                reward_schedules_creation_request_id.to_string(),
+            )
+            .add_attribute(
+                "request_sender",
+                pool_creation_request_context.request_sender.to_string(),
+            );
     }
 
+    event = event
+        .add_attribute(
+            "pool_creation_request_id",
+            pool_creation_request_id.to_string(),
+        )
+        .add_attribute("pool_id", pool_id.to_string())
+        .add_attribute("proposal_id", proposal_id.to_string())
```

Comment:
We can skip values that are already added to a previous event. For example, we can skip proposal_id here since by other events (post_proposal_creation_callback event in this case), we already have tracking that a particular proposal_id belongs to a particular pool_creation_request_id so this provides no new information to index.

Similarly, we can skip attributes like LP token, multistaking address since those are also duplicated and we can get them from the Pool table while indexing this event.